app/_includes/support/support-policy.md

```diff
@@ -1,3 +1,40 @@
+### Kong Mesh release policy
+
+Kong adopts a structured approach to versioning its products. Kong Mesh follow a pattern of {MAJOR}.{MINOR}.{PATCH}.
+
+{:.info}
+> **Long Term Support Policy Update**
+> <br><br>
+> Beginning in October 2025, we plan to release 4 minor versions per year, every year: one in January, one in April, one in July, and the last one in October. 
+> Each year, the first version we release will become an LTS release. 
+> Starting from 2.13, we will have 1 LTS release every year, in November* of that year.
```

Comment:
This contradicts the line above no (there's no minor version in Nov)? How about we shift 2.13 to Jan and say that the first release of the year will be LTS?